### PR TITLE
refactor(app): remove USB-to-Ethernet adapter serial number and IP info

### LINKED ISSUE #8438: App: "Remove the USB-to-Ethernet Adapter Info" Section

### PR DESCRIPTION
**Pull request by brenthagen**
# Overview

Closes https://github.com/Opentrons/opentrons/issues/8438
After some discussion, this is a reworking of the original PR https://github.com/Opentrons/opentrons/pull/8498

# Changelog

 - Removed USB-to-Ethernet serial number and IP information from the System Information Card
 - Updated design of the USB-to-Ethernet driver outdated warning

# Review requests

 - The three states of the USB-to-Ethernet info section:

no USB-to-Ethernet device:

Screen Shot 2021-10-13 at 3 20 39 PM

any adapter found (ignore "unknown", redux will populate the field as before):

Screen Shot 2021-10-13 at 3 16 07 PM

realtek adapter found and driver outdated:

Screen Shot 2021-10-13 at 3 15 37 PM

# Risk assessment

low

### CODE REVIEW

**Comment by emilywools** [comment]
Looks great!! thank you Brent

File: app/src/pages/More/__tests__/NetworkAndSystem.test.tsx [hunk 1]
```diff
@@ -1,14 +1,61 @@
...
+
+function stubSelector<R>(
+  mock: jest.MockedFunction<(s: State) => R>,
+  rVal: R
+): void {
+  mock.mockImplementation(state => {
+    expect(state).toBe(MOCK_STATE)
+    return rVal
+  })
+}
```
Comment (mcous) [inline app/src/pages/More/__tests__/NetworkAndSystem.test.tsx]: Oh hey, this pattern predates our inclusion of `jest-when`, which provides a _much_ better API for doing this.

```ts
import { when, resetAllWhenMocks } from 'jest-when'


// ...
beforeEach(() => {
  when(getU2EAdapterDevice).calledWith(MOCK_STATE)
    .mockReturnValue(null)
  
  when(getU2EWindowsDriverStatus).calledWith(MOCK_STATE)
    .mockReturnValue(SystemInfo.NOT_APPLICABLE)
})

afterEach(() => {
  resetAllWhenMocks()
})
```

**Review by mcous — APPROVED** [review]
Code LGTM! These tests predate `jest-when`, so there's a little cleanup opportunity if you're into that sort of thing 🧪

File: app/src/pages/More/SystemInfoCard/U2EDeviceDetails.tsx [hunk 2]
```diff
@@ -1,97 +1,75 @@
...
 
 export interface U2EDeviceDetailsProps {
   device: UsbDevice | null
-  ifaces: NetworkInterface[]
+  driverOutdated: boolean
 }
 
 const DetailText = styled.span`
-  min-width: 8rem;
+  min-width: ${SPACING_6};
```
Comment (b-cooper) [inline app/src/pages/More/SystemInfoCard/U2EDeviceDetails.tsx]: `SPACING_X` consts should be used for `padding` and `margin` values there are separate `SIZE_X` consts for `width`, `height` etc.   In this case `SIZE_4` should be equivalent

File: app/src/pages/More/SystemInfoCard/U2EDeviceDetails.tsx [hunk 3]
```diff
@@ -1,97 +1,75 @@
...
-  { label: 'Driver Version', property: 'windowsDriverVersion' },
-]
-
 const DetailItem = ({
   label,
   value,
 }: {
   label: string
-  value: string | number
+  // TODO(bh, 2021-10-12): may need to type react-i18next https://react.i18next.com/latest/typescript
```
Comment (b-cooper) [inline app/src/pages/More/SystemInfoCard/U2EDeviceDetails.tsx:32]: good call 👍

**Review by b-cooper — APPROVED** [review]
Looks good to me! 🧮

**Pull request merged**